### PR TITLE
MOSIP-29294 Audit call request changes done

### PR DESCRIPTION
**Pull request by Neha2365**

### CODE REVIEW

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java [hunk 1]
```diff
@@ -340,4 +359,20 @@ public void updateCredentialStatus(CredentialStatusEvent credentialStatusEvent)
...
+				securityManager.hash(id.getBytes()), idType, desc);
+		HttpEntity<RequestWrapper<AuditRequestDTO>> httpEntity = new HttpEntity<>(auditRequest);
+		try {
+			restUtil.postApi(ApiName.KERNELAUDITMANAGER, null, id, desc, MediaType.APPLICATION_JSON, httpEntity,
+					AuditResponseDTO.class);
+		} catch (IdRepoDataValidationException e) {
+			LOGGER.error(IdRepoSecurityManager.getUser(), "AuditRequestBuilder", "audit",
+					"Exception : " + ExceptionUtils.getStackTrace(e));
+		} catch (Exception e) {
+			LOGGER.error(IdRepoSecurityManager.getUser(), "AuditRequestBuilder", "audit",
```
Comment (vishwa-vyom) [inline id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java:373]: I think for this and above, "AuditRequestBuilder" should be "CredentialRequestServiceImpl"

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
  */
 public enum ApiName {
 
-	CRDENTIALSERVICE;
+	CRDENTIALSERVICE,
+	KERNELAUDITMANAGER;
```
Comment (vishwa-vyom) [inline id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java:11]: this also need the config to be added in the property files ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java (hunk 1 above)
Comment (Neha2365) [reply id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java]: Resolved

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java (hunk 2 above)
Comment (Neha2365) [reply id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java:11]: It is added in property file in mosip-config branch qa3-1.1.5

**Review by vishwa-vyom — APPROVED** [review]

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java (hunk 2 above)
Comment (vishwa-vyom) [inline id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java:11]: There is already a property called "mosip.idrepo.audit.rest.uri",  lets try to use that so if someone uses this new change they don't need to add new properties.

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java (hunk 2 above)
Comment (Neha2365) [reply id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java]: Changes done

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**